src/core/server/core_usage_data/core_usage_stats_client.ts

```diff
@@ -88,67 +105,124 @@ export class CoreUsageStatsClient {
+    const counterFieldNames = [`createNewCopiesEnabled.${createNewCopies ? 'yes' : 'no'}`];
+    await this.updateUsageStats(counterFieldNames, RESOLVE_IMPORT_STATS_PREFIX, options);
   }
 
-  public async incrementSavedObjectsExport({
-    headers,
-    types,
-    supportedTypes,
-  }: IncrementSavedObjectsExportOptions) {
-    const isKibanaRequest = getIsKibanaRequest(headers);
+  public async incrementSavedObjectsExport(options: IncrementSavedObjectsExportOptions) {
+    const { types, supportedTypes } = options;
     const isAllTypesSelected = !!types && supportedTypes.every((x) => types.includes(x));
-    const counterFieldNames = [
-      'total',
-      `kibanaRequest.${isKibanaRequest ? 'yes' : 'no'}`,
-      `allTypesSelected.${isAllTypesSelected ? 'yes' : 'no'}`,
-    ];
-    await this.updateUsageStats(counterFieldNames, EXPORT_STATS_PREFIX);
+    const counterFieldNames = [`allTypesSelected.${isAllTypesSelected ? 'yes' : 'no'}`];
+    await this.updateUsageStats(counterFieldNames, EXPORT_STATS_PREFIX, options);
   }
 
-  private async updateUsageStats(counterFieldNames: string[], prefix: string) {
+  private async updateUsageStats(
+    counterFieldNames: string[],
+    prefix: string,
+    { request }: BaseIncrementOptions
+  ) {
     const options = { refresh: false };
     try {
       const repository = await this.repositoryPromise;
+      const fields = [...this.getCommonFieldsToIncrement(request), ...counterFieldNames];
       await repository.incrementCounter(
         CORE_USAGE_STATS_TYPE,
         CORE_USAGE_STATS_ID,
-        counterFieldNames.map((x) => `${prefix}.${x}`),
+        fields.map((x) => `${prefix}.${x}`),
         options
       );
     } catch (err) {
       // do nothing
     }
   }
+
+  private getIsDefaultNamespace(request: KibanaRequest) {
+    const requestBasePath = this.basePath.get(request); // obtain the original request basePath, as it may have been modified by a request interceptor
+    const pathToCheck = this.basePath.remove(requestBasePath); // remove the server basePath from the request basePath
+    const matchResult = pathToCheck.match(SPACE_CONTEXT_REGEX); // Look for `/s/space-url-context` in the base path
+
+    if (!matchResult || matchResult.length === 0) {
+      return true;
+    }
+
+    // Ignoring first result, we only want the capture group result at index 1
+    const [, spaceId] = matchResult;
+
+    return spaceId === DEFAULT_NAMESPACE_STRING;
+  }
+
+  private getCommonFieldsToIncrement(request: KibanaRequest) {
+    const isKibanaRequest = getIsKibanaRequest(request);
+    const isDefaultNamespace = this.getIsDefaultNamespace(request);
+    const namespaceField = isDefaultNamespace ? 'default' : 'custom';
+    const counterFieldNames = [
+      'total',
+      `namespace.${namespaceField}.total`,
+      `namespace.${namespaceField}.kibanaRequest.${isKibanaRequest ? 'yes' : 'no'}`,
+    ];
+    return counterFieldNames;
+  }
+}
+
+function getAllCommonFields(prefix: string) {
```

Comment:
NIT: how are these fields 'common'? Wouldn't `getFieldsForCounter` or something like that be more explicit? (same for `getCommonFieldsToIncrement`)